place.project.json

```diff
@@ -11,7 +11,7 @@
       },
 
       "TestEZ": {
-        "$path": "modules/testez/lib"
+        "$path": "modules/testez"
```

Comment:
Why does this not need to path to modules/testez/src, like in bin/spec.lua? Does it just automatically load the src?

Reply:
When you give a path to a directory to Rojo, it will look for a file named `default.project.json` and use that to map the directory to instances. If it does not exist, it'll default to just the regular file to instance mapping rules.

I think this file was probably written before Rojo was able to do that